src/components/shared/FilterNavigatorBar.css

```diff
@@ -125,25 +132,30 @@
 
 /* Animation */
 
+.filterNavigatorBarUncommittedTransition-exit {
+  /* Because of the underlying transition library, this element is still here
+   * while the new "committed" element is created, which pushes it further
+   * right. By using display: none here, we prevent this bad effect. */
+  display: none;
+}
+
 .filterNavigatorBarTransition-enter {
-  z-index: 0;
-  opacity: 0.1;
-  transform: translateX(-100%);
+  color: inherit;
+  opacity: 0.65;
 }
 
 .filterNavigatorBarTransition-enter.filterNavigatorBarTransition-enter-active {
+  color: var(--internal-selected-color);
   opacity: 1;
-  transform: translateX(0);
-  transition: opacity 300ms ease-out, transform 300ms ease-out;
+  transition: color 300ms, opacity 300ms;
 }
 
 .filterNavigatorBarTransition-exit {
   opacity: 1;
-  transform: translateX(0);
 }
 
 .filterNavigatorBarTransition-exit.filterNavigatorBarTransition-exit-active {
-  opacity: 0.01;
+  opacity: 0.1;
```

Comment:
I recall an old recommandation to start with something else than "0" so that the browser is "prepared" to transition the value. Not sure if this is still a current recommendation...

Incidentally, you were the one introducing these non-0 values in the first place:
https://github.com/firefox-devtools/profiler/pull/203/files#diff-aee0e1476f1df6298039fd72b697af570dc199d0241d35940dff82a98f9a2acbR114
https://github.com/firefox-devtools/profiler/pull/203/files#diff-aee0e1476f1df6298039fd72b697af570dc199d0241d35940dff82a98f9a2acbR131
So maybe you used to follow this recommendation too ? :-)

Reply:
It looks like that the old documention for react-transition-group used `0.01`, but the current doc actually uses `0`. So something changed. However their documentation isn't in git (or at least I haven't found it) so I can't find the rationale behind their change.

Reply:
I changed all of these odd values to 0.